src/3id/orbitProvider.js

```diff
@@ -0,0 +1,42 @@
+// const Identities = require('orbit-db-identity-provider')
+const IdentityProvider = require('orbit-db-identity-provider/src/identity-provider-interface.js')
+
+
+class OrbitIdentityProvider {
+  constructor (options={}) {
+    // super(options)
+    console.log('create new identity')
+    console.log(options)
+    this.pubKey = options.pubKey
+    console.log('pubkey')
+    console.log(this.pubKey)
+  }
+
+  static get type () { return '3ID' } // return type
+  // return identifier of external id (eg. a public key)
+  async getId () {
```

Comment:
Looks like right now we are creating a separate provider for each identity/space. Is this desired? Another options would be for the `getId` and `signIdentity` function to accept the `options` parameter and include the space name in there. This options can be passed to the `createIdentity` [function](https://github.com/orbitdb/orbit-db-identity-provider/blob/master/src/identities.js#L36).